server/config/validate-config-keys.js

```diff
@@ -46,7 +46,9 @@ const sharedConfig = parseConfig( '_shared.json' );
  * invalid. Such a missing value could and likely would
  * cause runtime errors in Calypso
  */
-environmentKeys.forEach( ( [ filename, keys ] ) => {
```

Comment:
It looks like your problems are because the GUI is using an older version of `node` (which doesn't support arguments destructuring), which means that somehow your `PATH` is not configured the same way than when you open a Terminal. this is a very fragile solution, because people will keep happily adding code in `Node v6`-compatible syntax, and that would break it for you.

Maybe we should dig into what's causing this instead, and add the solution to `docs/troubleshooting.md`.